parser.go

```diff
@@ -5,13 +5,18 @@ import (
 	"strconv"
 )
 
-// parser provides a simple way of accessing and parsing
+// Parser provides a simple way of accessing and parsing
 // sentence fields
 type parser struct {
```

Comment:
Since we are exposing the parser to external users we should now export it.

The linter is also complaining about it.
```
parser.go:16:32: exported func NewParser returns unexported type *nmea.parser, which can be annoying to use
```

Reply:
wasnt sure if it was like this on purpose, done